scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -419,7 +419,7 @@ abstract class FixedPathSource(path: String*) extends FileSource {
   def localPaths = path.toList
   def hdfsPaths = path.toList
 
-  override def toString = getClass.getName + path
+  override def toString = getClass.getName + path.mkString(",")
```

Comment:
@ianoc Before this change, the toString would generate MultipleTextLineFilesListinputname for "inputname" path. It doesn't match when JobTest is given a source, which outputs MultipleTextLineFilesWrappedArrayinputname